src/site/content/en/handbook/content-checklist/index.md

```diff
@@ -27,19 +27,19 @@ This is a checklist for web.dev content reviews. It includes critical high-level
 ## Instruction
 1. If the piece expects readers to have prior knowledge, is that made clear up front?
 1. Does the piece explain why the process being taught matters?
-1. Are examples provided to support all key claims and ideas?
-1. Are instructions in second person (_you/your_), not first person (_we/our_)?
-1. Are common instructions provided using the web.dev [Instruction components](/handbook/web-dev-components/#instruction)?
+1. Do the examples support all key claims and ideas?
+1. Are instructions written in second person (_you/your_), not first person (_we/our_)?
+1. Do the common instructions follow the web.dev [Instruction components](/handbook/web-dev-components/#instruction)?
```

Comment:
```suggestion
1. Are common instructions included using the web.dev [Instruction components](/handbook/web-dev-components/#instruction)?
```
Again, the meaning was changed. Common instructions aren't following a template; rather writers should use a shortcode so the instructions are populated by the component.